client/src/schemas/material.js

```diff
@@ -0,0 +1,156 @@
+import { object, string, array, boolean } from 'yup'
+
+export const categories = [
+  'CELL_LINE',
+  'PLASMID',
+  'PROTOCOL',
+  'DATASET',
+  'MODEL_ORGANISM',
+  'PDX',
+  'OTHER'
+]
+
+export const importSources = [
+  'GEO',
+  'SRA',
+  'DBGAP',
+  'DATASET',
+  'PROTOCOLS_IO',
+  'ADDGENE',
+  'JACKSON_LAB',
+  'ATCC',
+  'ZIRC',
+  'OTHER'
+]
+
+const materialCategories = {
+  CELL_LINE: object({
+    cell_line_name: string(),
+    tissue: string(),
+    disease: string(),
+    number_of_availible_cell_lines: string(),
+    age: string(),
+    sex: string(),
+    ethnicity: string(),
+    bio_safety_level: string(),
+    population_doubling_time: string(),
+    storage_medium: string(),
+    growth_medium: string(),
+    subculturing: string(),
+    cryopreservation: string(),
+    culture_condition: string()
+  }),
+  PLASMID: object({
+    plasmid_name: string(),
+    plasmid_type: string(),
+    purpose: string(),
+    number_of_available_samples: string(),
+    bio_safety_leve: string(),
+    gene_insert_name: string(),
+    relevant_mutations: string(),
+    backbone_name: string(),
+    primary_vector_type: string(),
+    cloning_method: string(),
+    bacterial_resistance: string(),
+    copy_number: string(),
+    growth_temp_celsius: string(),
+    growth_strain: string()
+  }),
+  PROTOCOL: object({
+    description: string(),
+    abstract: string(),
+    protocol_name: string()
+  }),
+  DATASET: object({
+    description: string(),
+    number_samples: string(),
+    technology: string(),
+    platform: string()
+  }),
+  MODEL_ORGANISM: object({
+    description: string(),
+    genetic_background: string(),
+    zygostity: string(),
+    number_of_available_models: string(),
```

Comment:
This shouldn't be an int?

Reply:
When I talked it over with deepa, we thought units should be included which makes leaving it a string a good idea.